File: KingdomCodeTool/wwwroot/Download/202503250844/Service/Implement/NoiTru_LuuTruChiTietService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class NoiTru_LuuTruChiTietService : BaseService<NoiTru_LuuTruChiTiet, INoiTru_LuuTruChiTietRepository>
    , INoiTru_LuuTruChiTietService
    {
    private readonly INoiTru_LuuTruChiTietRepository _NoiTru_LuuTruChiTietRepository;
    public NoiTru_LuuTruChiTietService(INoiTru_LuuTruChiTietRepository NoiTru_LuuTruChiTietRepository) : base(NoiTru_LuuTruChiTietRepository)
    {
    _NoiTru_LuuTruChiTietRepository = NoiTru_LuuTruChiTietRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202503250844/Angular/shared/NoiTru_LuuTruChiTiet.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NoiTru_LuuTruChiTiet } from './NoiTru_LuuTruChiTiet.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NoiTru_LuuTruChiTietService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'LuuTruChiTiet_Id', 'LuuTru_Id', 'PhongBan_Id', 'GiuongBenh_Id', 'PhongBenh_Id', 'NgayVao', 'ThoiGianVao', 'ThoiGianTinhLuuTru', 'LyDo_Id', 'NgayRa', 'ThoiGianRa', 'DonGiaPhaiThu', 'DonGia', 'TyLeVAT', 'GiaTriVAT', 'TienTe_Id', 'Tygia', 'LoaiGia_Id', 'GiuongBenhChuyenDen_Id', 'BaoPhong', 'KhongTinhTien', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id', 'LuuTruChiTiet_Prev', 'LuuTruChiTiet_Current', 'DoiTuong_Id', 'LuuTruChiTiet_Id_Last', 'SiteCode', 'Mapping_Public_Id'];

    List: NoiTru_LuuTruChiTiet[] | undefined;
    ListFilter: NoiTru_LuuTruChiTiet[] | undefined;
    FormData!: NoiTru_LuuTruChiTiet;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NoiTru_LuuTruChiTiet";
    }
}

